frontends/web/tests/routes/account/send/__snapshots__/feetargets.test.tsx.snap

```diff
@@ -1,57 +1,55 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`routes/account/send/feetargets should match the snapshot 1`] = `
-preact-render-spy (1 nodes)
--------
 <div
-  class="input align-left  isTransparent"
+  className="   "
```

Comment:
need to investigate

Reply:
I can't get typed-css-modules to work in enzyme, but if we need to test those classnames we can `expect(context.find('.input').hasClass('align-left')).toBe(true);`